ProcessLib/HeatTransportBHE/BHE/BHECommonCoaxial.cpp

```diff
@@ -130,6 +131,25 @@ BHECommonCoaxial::calcThermalResistances(double const Nu_inner_pipe,
     return getThermalResistances(R_gs, R_ff, R_fg);
 }
 
+std::array<std::pair<std::size_t /*node_id*/, int /*component*/>, 2>
+BHECommonCoaxial::getBHEInflowDirichletBCNodesAndComponents(
+    std::size_t top_node_id,
+    std::size_t /*bottom_node_id*/,
+    int in_component_id)
```

Comment:
For safety, this arguments and the function should be `const`.

Reply:
Added.